Documentation/install/get-dotnetcore-dnx-linux.md

```diff
@@ -64,8 +64,8 @@ You can see the currently installed DNX versions with `dnvm list`.
 ```
 Active Version              Runtime Arch Location             Alias
 ------ -------              ------- ---- --------             -----
-  *    1.0.0-beta5-11649    coreclr x64  ~/.dnx/runtimes
-       1.0.0-beta5-11649    mono         ~/.dnx/runtimes      default
+  *    1.0.0-beta7-12364    coreclr x64  ~/.dnx/runtimes
+       1.0.0-beta7-12364    mono         ~/.dnx/runtimes      default
```

Comment:
Would be good to add a note here saying something to the effect of "your display may vary as new versions of the DNX are published" or similar.